### ISSUE TITLE
Debounce not working as expected

### ISSUE THREAD

**Issue by CyberCyclone** [robfallows/tunguska-reactive-aggregate#27]
I'm unsure on how the debounce function is supposed to work, specifically the debounceDelay. The way it's setup, it only runs once and never again. Shouldn't it clear the `timer` once it's done with it so it can be used again? Here's a code snippet of how I'm expecting it should work, specifically setting timer as `undefined`:

```javascript
    const debounce = (notification) => {
        if (initializing) return;
        if (localOptions.debug) console.log(`Reactive-Aggregate: collection: ${notification.name}: publish: 
        ${notification.mutation}, _id: ${notification.id}`)
        if (!timer && localOptions.debounceCount > 0) {
            timer = Meteor.setTimeout(function(){
                // We need to clear the timer otherwise it will never run again.
                timer = undefined;
                update();
            }, localOptions.debounceDelay);
        }
        
        if (++currentDebounceCount > localOptions.debounceCount) {
            currentDebounceCount = 0;
            Meteor.clearTimeout(timer);
            timer = undefined;
            update();
        }
    }
```

If this is a bug and the above code looks correct then I can create a pull request. Based on my understanding the above code works in my situation.

**Comment by robfallows** [comment]
Agreed. I found another typo in the code, so I've fixed this and that in v1.2.5.

**Comment by CyberCyclone** [comment]
Awesome, thanks for that. I've swapped back to the Meteor package (from using a local version) and the changes you've made seem to be working correctly.
